packages/grafana-ui/src/components/ColorPicker/ColorSwatch.tsx

```diff
@@ -40,6 +50,8 @@ export const ColorSwatch = React.forwardRef<HTMLDivElement, Props>(
       swatchStyles.border = `2px solid ${theme.colors.border.medium}`;
     }
 
+    const colorLabel = `${label || ariaLabel} color`;
```

Comment:
We're using the label over the ariaLabel?

Reply:
great catch! would push some changes now